__tests__/util.test.ts

```diff
@@ -162,3 +164,33 @@ test("getClubhouseURLFromPullRequest comment", async () => {
 
   scope.done();
 });
+
+test("shouldProcessPullRequestForUser user lists not defined", async () => {
+  expect(util.shouldProcessPullRequestForUser("github-author-1")).toBeTruthy;
+});
+
+test("shouldProcessPullRequestForUser both user lists defined", async () => {
+  process.env["INPUT_ONLY-USERS"] = "fake-user-1";
+  process.env["INPUT_IGNORED-USERS"] = "fake-user-1";
+  expect(util.shouldProcessPullRequestForUser("fake-user-1")).toBeFalsy;
+});
+
+test("shouldProcessPullRequestForUser fake-user-1 in only-users", async () => {
+  process.env["INPUT_ONLY-USERS"] = "fake-user-1";
+  expect(util.shouldProcessPullRequestForUser("fake-user-1")).toBeTruthy;
+});
+
+test("shouldProcessPullRequestForUser fake-user-1 in ignored-users", async () => {
+  process.env["INPUT_IGNORED-USERS"] = "fake-user-1";
+  expect(util.shouldProcessPullRequestForUser("fake-user-1")).toBeFalsy;
```

Comment:
All of these `.toBeFalsy` and `.toBeTruthy` calls need parentheses, otherwise they won't be actually _called_. For example:
```suggestion
  expect(util.shouldProcessPullRequestForUser("fake-user-1")).toBeFalsy();
```
See [the Jest docs on matchers](https://jestjs.io/docs/en/using-matchers).

Reply:
Yea, my bad. Thank you for the help and documentation.